docs/3.0.0-beta.x/community/contribute-with-docs.md

```diff
@@ -0,0 +1,486 @@
+
+Examples include [Why we split the management of the Admin user and the Front-end Users](https://blog.strapi.io/why-we-split-the-management-of-the-admin-users-and-end-users/) and [Restricting Permissions in Strapi Admin](https://blog.strapi.io/admin-permissions/).
+
+These four categories follow the recommendations found in this article about [documentation by Divio](https://www.divio.com/blog/documentation/). We recommend reading it to gain a complete understanding of each type of content contained in our documentation.
+
+## The Workflow
+
+Please read the following sections before moving to create documentation. (Either new documentation or modifying existing documentation.)
+
+1. [Technical Requirements and the Installation of the Docs](/3.0.0-beta.x/community/contribute-with-docs.html#technical-requirements)
+2. [Understanding the Docs Structure](/3.0.0-beta.x/community/contribute-with-docs.html#understanding-the-docs-structure)
+3. [Submitting new documentation](/3.0.0-beta.x/community/contribute-with-docs.html#submitting-totally-new-documentation)
+4. [Adding/modifying/deleting documentation](/3.0.0-beta.x/community/contribute-with-docs.html#adding-modifying-deleting-content)
+
+- How to [submit an externally hosted tutorial](/3.0.0-beta.x/community/contribute-with-docs.html#submitting-an-externally-hosted-tutorial).
+- How to use the [Docs Style Guide](/3.0.0-beta.x/community/contribute-with-docs.html#docs-style-guide) to write better Docs.
+
+### Technical Requirements
+
+1. You have [Node v10.x.x](https://nodejs.org/en/download/) (LTS) or [Node v12.x.x](https://nodejs.org/en/download/current/) and [Yarn v1.2.0+](https://yarnpkg.com/en/).
+2. You are familiar with Git, and have it installed on your development environment.
+
+- Ensure you're using the [required versions of Node.js and npm](/3.0.0-beta.x/getting-started/install-requirements.html).
+- Fork the [Strapi repository](https://github.com/strapi/strapi) [to your own GitHub account](https://help.github.com/en/articles/fork-a-repo).
+- Clone your Strapi repository:
+
+`Path: ./Projects/`
+
+```bash
+git clone git@github.com:YOUR_USERNAME/strapi.git
+```
+
+- Install the dependencies **for the docs** section of the monorepo:
+
+`Path: ./Projects/`
+
+```bash
+cd strapi/docs
+npm install
+```
+
+- Start the docs development server and open the documentation in the browser:
+
+```bash
+npm run dev
+```
+
+- Then, navigate to [http://localhost:8080/documentation/](http://localhost:8080/documentation/).
+
+You should now have a running instance of the documentation. The next section explains the organization and structure of the documentation.
+
+### Understanding the Docs Structure
+
+The Strapi docs are power by [VuePress](https://vuepress.vuejs.org/). The docs live in a folder called **docs**. `Path: ./strapi/docs`. The _VuePress_ installation lives in this folder. It has a separate installation and separate dependencies from the rest of the Strapi monorepo.
```

Comment:
The Strapi docs are powered by [...] monorepository

Reply:
I changed monorepo to monorepository.